src/event.js

```diff
@@ -0,0 +1,50 @@
+import {
+  addItem,
+  removeItem,
+  displayToDos,
+  getDescriptionInput,
+  markCompleted,
+  clearMethod,
+} from './index.js';
```

Comment:
🍾  
Nice use of DRY principles here including all methods under one import declaration. This makes reading the code easier and improves maintainability.